lib/publiccloud/utils.pm

```diff
@@ -54,7 +54,8 @@ sub register_addon {
     my ($remote, $addon) = @_;
     my $arch = get_var('PUBLIC_CLOUD_ARCH') // "x86_64";
     $arch = "aarch64" if ($arch eq "arm64");
-    record_info($addon, "Going to register '$addon' addon");
+    my $timestamp = script_output("date -u +%d/%m/%Y%t%H:%M:%S");
```

Comment:
```perl
use DateTime;
```
```suggestion
    my $dt = DateTime->now;
    my $timestamp = sprintf("%s. %s. %s:%s", $dt->day, $dt->month, $dt->hour, $dt->minute);
```

Reply:
That means would be taking the local time of the worker, which is not guaranteed to be UTC. I don't like this.

Reply:
I changed it to use `gmtime`.